components/map/basic/src/leaflet/map.js

```diff
@@ -162,7 +163,123 @@ export default class LeafletMap {
     }
   }
 
+  pnpoly(x, y, coords) {
```

Comment:
Is this magic? What's this doing?

Reply:
I don't know 🤣, this was copy-paste from here https://github.com/maxogden/geojson-js-utils/blob/aa31043f31cb65d573a29dfc9faef49f12702b7c/geojson-utils.js#L72